src/cluster_legacy.c

```diff
@@ -6911,6 +6909,11 @@ int clusterCommandSpecial(client *c) {
             cport = port + CLUSTER_PORT_INCR;
         }
 
+        if (cport <= 0 || cport > 65535) {
+            addReplyErrorFormat(c, "Cport number is out of range");
```

Comment:
Just thinking about this error message. In the docs, we called it cluster-bus-port:

    CLUSTER MEET ip port [cluster-bus-port]

And in the other error for this field uses the word "bus port", when getLongLongFromObject fails:

    addReplyErrorFormat(c, "Invalid bus port specified: %s", (char *)c->argv[4]->ptr);

What do you think about this:

```suggestion
            addReplyErrorFormat(c, "Cluster bus port number is out of range");
```

Reply:
Agree, we should match with the doc.